inference-engine/ie_bridges/java/samples/benchmark_app/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2020 Intel Corporation
+
+get_filename_component(sample_name "${CMAKE_CURRENT_SOURCE_DIR}" NAME)
+
+file(GLOB_RECURSE src ${CMAKE_CURRENT_SOURCE_DIR}/*.java)
+list(APPEND src ${PROJECT_SOURCE_DIR}/samples/ArgumentParser.java)
```

Comment:
Seems like we do the same logic for every sample. Let's move to the function

Reply:
Done